src/current/v23.1/security-reference/authorization.md

```diff
@@ -134,6 +134,7 @@ Users that own objects cannot be dropped until the [ownership is transferred to
 
 When a user connects to a database, either via the built-in SQL client or a client driver, CockroachDB checks the user and role's privileges for each statement executed. If the user does not have sufficient privileges for a statement, CockroachDB gives an error.
 
+<a id="role-options"></a>
 ### Supported privileges
 
 System-level privileges (also known as global privileges) offer more granular control over a user's actions when working with CockroachDB, compared to the [role options authorization model](#role-options).
```

Comment:
The anchor link for role options: "compared to the [role options authorization model](#role-options)." (line 140) just takes the reader back to where they are.

Ah, reading through Rafi's comments, it seems like maybe the anchor link should go after the table, there is a paragraph there that refers to role options

Reply:
Ahh I see, I just need to unlinkify the link in line 140. The anchor is to take people who have bookmarked the now-removed section to the Supported privileges section instead.

Reply:
@rafiss PTAL at the current state of the content around line 140 here. I reworked the paragraph to avoid needing to link to the "Role Options" section we just removed. I am not sure what the difference is between supported privileges and role options -- do you think maybe the fact that they contained different info wasn't a duplication?